doc/release-notes/9260-solr930.md

```diff
@@ -0,0 +1,3 @@
+Solr has been upgraded to Solr 9. You must install Solr fresh and reindex. 
+
+If you have custom metadata blocks installed, you must update your Solr schema.xml to include your custom fields. Please see https://guides.dataverse.org/en/9260-solr930/admin/metadatacustomization.html#updating-the-solr-schema
```

Comment:
Any chance we could add just a couple of sentences here, clarifying what needs to be done? I fear this may confuse some people. Instead of `you must update your solr schema`, maybe spell it out as `you must copy the schema.xml from the source tree, here, then update it to include your custom fields, see ... for instructions. If you don't use custom metadata blocks, use the schema.xml supplied in the source tree as is. **Do not attempt to reuse your existing 8.11 schema.xml!**` - something along these lines?

Reply:
@landreev sure, good idea. Please check out be34440132 and please feel free to edit at will!